### DISCUSSION TITLE
Codeeya dhalinyaro

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#12] [Polls]
@dugsiiyeinc/dhaqsopay-extension-developers 
@dugsiiyeinc/dhaqsopay-server-developers 
@dugsiiyeinc/dhaqsopay-server-developers 

1aad:
https://preview.themeforest.net/item/raveli-online-payment-app-elementor-template-kit/full_screen_preview/43419505

2aad:
https://html.designingmedia.com/repay/?storefront=envato-elements

**Comment by sharafdin** [comment]
labadaba midkaan qadano ma xumo lakin aniga ahaan midka 1aad ayaa simple and pro ila ah.

**Reply by Nasrah-muse** [reply]
dabcan eng ani xitaa kaas baa ila shidan oo simple ah  wxii qurxin ah dhib maleh waa lagu dari karaa

**Comment by hayle01** [comment]
Aniga labadaba uma codeyn kuwo kale in la fiiriyo ayaa qabaa oo la raadiyo

**Reply by sharafdin** [reply]
> Aniga labadaba uma codeyn kuwo kale in la fiiriyo ayaa qabaa oo la raadiyo

maxumo eng lkn waqti ayaa nagu socda talaadada in aan kaga dhamaano ayaa la rabaa hadii aad mid kale heleysid nala so wadaag hadi kale mid u codee Insha'Allah!

**Comment by sharafdin** [comment]
@Hayle01 eng mid makaa sugnaa mise kan sare waan ku dhaqaaqnaa ?
